### DISCUSSION TITLE
Using two PCs to finish export faster?

### DISCUSSION THREAD

**Discussion by Ryudoadema** [nagadomi/nunif#172] [Q&A]
I am trying to find a way to batch edit image numbers in order, but maybe with your knowledge on the inner workings you know of another way within iw3 to accomplish this? I am trying to use two separate PCs to speed up my dmap export from iw3 that will otherwise take 3+ days. I start one partway through and once every frame is done between the two with some overlap, I can copy-paste from one to the other starting at the exact correct frame. Only issue I can think of is the numbering...

**Comment by Ryudoadema** [comment]
Nevermind I'm stupid. Iw3 already names the frame correctly no matter where you start. You rock nagadomi!

**Comment by nagadomi** [comment]
Not related to the title,
Did you set `Worker Threads` to 16? This setting is a very important setting for performance of Export.
Also, using SSD for output directory should help improve performance.
Export has very high disk IO load compared to normal SBS convertion. others parts should be faster than normal SBS conversion, because the stereo generation process is skipped.

**Comment by Ryudoadema** [comment]
I am exporting to an ssd. I tried using 16 worker threads when I saw one of your previous posts about it. I found that it starts out much faster for a few seconds, then over the course of a couple minutes actually slowed down below what a lower worker thread setting is stable at.

Edit: Just tried 16 again and once it got through the previously exported images it seems to be the same speed. Any chance we can have it tell us the wait time average from restart instead of counting the already exported images at 1000fps or whatever it is? If you ever stop and restart to try and find faster settings or after restarting PC, you no longer have an accurate gauge on time remaining.

**Reply by nagadomi** [reply]
It is a resume issue.
The remaining time can be more accurate if processing FPS is calculated from the past few minutes, but in anyway it is better not to use resume for benchmarking.

**Reply by Ryudoadema** [reply]
> It is a resume issue. The remaining time can be more accurate if processing FPS is calculated from the past few minutes, but in anyway it is better not to use resume for benchmarking.

Maybe a pause on calculating would be a way to fix that? Let it wait a bit to start calculating, that way it can still use the time before for an average. But only starting a minute or two in. And only for if it is resuming, then people can still benchmark on new exports.
